addons/website_event_sale/views/website_sale_templates.xml

```diff
@@ -41,44 +41,114 @@
-                                        Download Tickets <i class="ms-1 fa fa-download"/>
+                                <div class="col-12 mb-2 d-flex flex-column flex-md-row align-items-md-baseline justify-content-md-center">
+                                    <a class="btn btn-primary" title="Download All Tickets" target="_blank" style="min-width: 9rem;"
+                                        t-attf-href="/event/{{ event.id }}/my_tickets?registration_ids={{ attendees.ids }}&amp;tickets_hash={{ event._get_tickets_access_hash(attendees.ids) }}">
+                                        Download <i class="ms-1 fa fa-download"/>
                                     </a>
-                                    <a class="mb-2" t-attf-href="/event/#{ slug(event) }">Go to Event</a>
+                                    <span class="mx-3">or</span>
+                                    <div class="o_send_by_email_widget has-validation ms-md-2 mt-2 mt-md-0" style="max-width: 20rem;">
+                                        <div class="input-group">
+                                            <input id="o_send_by_email_input" type="text"
+                                                placeholder="john@doe.com" class="form-control"/>
+                                            <button class="o_wevent_js_send_email btn btn-primary"
+                                                type="button"
+                                                t-att-data-event-id="event.id"
+                                                t-att-data-attendees-ids="attendees.ids"
+                                                t-att-data-tickets-hash="event._get_tickets_access_hash(attendees.ids)">
+                                                Send by Email
+                                            </button>
+                                        </div>
+                                        <div class="invalid-feedback">
+                                            Please enter valid email(s)
+                                        </div>
+                                    </div>
+                                    <div class="o_send_by_email_info alert alert-info ms-md-2 mt-2 mt-md-0 mb-0 px-3 py-1 d-inline-flex align-items-center justify-content-center d-none" role="alert">
+                                        <span>
+                                            <strong>See you soon!</strong> Your tickets have been sent.
+                                        </span>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="row mb-3 o_wereg_confirmed_attendees">
+                                <div class="col-md-4 col-xs-12 mt-3" t-foreach="attendees" t-as="attendee">
```

Comment:
Mmh? double check variable scope for `attendees` please, it may be the last one of the loop in the previous div? Should it be all attendees or the ones of the event? In any case, something feels strange here ^^

Reply:
Not sure to understand what you mean here :thinking:

Reply:
Put the t-set of attendees just after the for loop of the event